$MirOS: ports/www/php/patches/patch-configure_in,v 1.3 2006/01/25 17:58:52 tg Exp $
$OpenBSD: patch-configure_in,v 1.1.1.1 2004/10/02 11:32:37 robert Exp $
--- configure.in.orig	Wed Jan 25 18:00:12 2006
+++ configure.in	Wed Jan 25 18:54:16 2006
@@ -31,9 +31,7 @@ AC_PREREQ(2.13)
 AC_INIT(README.CVS-RULES)
 
 PHP_CONFIG_NICE(config.nice)
-for arg in $0 "$@"; do
-  CONFIGURE_COMMAND="$CONFIGURE_COMMAND '$arg'"
-done
+CONFIGURE_COMMAND="Built via the MirPorts Framework"
 
 PHP_CANONICAL_HOST
 AC_CONFIG_HEADER(main/php_config.h)
@@ -447,7 +445,7 @@ PHP_TM_GMTOFF
 PHP_STRUCT_FLOCK
 PHP_SOCKLEN_T
 
-AC_CHECK_SIZEOF(intmax_t, 0)
+AC_CHECK_SIZEOF(intmax_t, 0, AC_INCLUDES_DEFAULT)
 AC_CHECK_SIZEOF(size_t, 8)
 AC_CHECK_SIZEOF(ssize_t, 8)
 AC_CHECK_SIZEOF(ptrdiff_t, 8)
@@ -505,6 +503,7 @@ dnl ------------------------------------
 AC_FUNC_VPRINTF
 AC_CHECK_FUNCS(
 alphasort \
+arc4random \
 asctime_r \
 chroot \
 ctime_r \
@@ -888,17 +887,14 @@ enable_static=yes
 
 case $php_build_target in
 program|static)
-    standard_libtool_flag='-prefer-non-pic -static'
+    standard_libtool_flag='-static'
     if test -z "$PHP_MODULES"; then
         enable_shared=no
     fi
 ;;
 shared)
     enable_static=no
-    case $with_pic in
-      yes) standard_libtool_flag='-prefer-pic';;
-      no)  standard_libtool_flag='-prefer-non-pic';;
-    esac
+    standard_libtool_flag=
     EXTRA_LDFLAGS="$EXTRA_LDFLAGS -avoid-version -module"
 ;;
 esac
@@ -1082,7 +1078,7 @@ EXPANDED_SYSCONFDIR=`eval echo $sysconfd
 EXPANDED_DATADIR=$datadir
 EXPANDED_PHP_CONFIG_FILE_PATH=`eval echo "$PHP_CONFIG_FILE_PATH"`
 EXPANDED_PHP_CONFIG_FILE_SCAN_DIR=`eval echo "$PHP_CONFIG_FILE_SCAN_DIR"`
-INCLUDE_PATH=.:$EXPANDED_PEAR_INSTALLDIR
+INCLUDE_PATH=.:$libdir
 
 exec_prefix=$old_exec_prefix
 libdir=$old_libdir
